packages/amplify-category-api/src/__tests__/provider-utils/awscloudformation/utils/rds-utils.test.ts

```diff
@@ -1,9 +1,25 @@
+import { DataSourceStrategiesProvider, ModelDataSourceStrategy } from '@aws-amplify/graphql-transformer-interfaces';
+import { DDB_DEFAULT_DATASOURCE_STRATEGY, MYSQL_DB_TYPE } from '@aws-amplify/graphql-transformer-core';
 import { checkForUnsupportedDirectives } from '../../../../provider-utils/awscloudformation/utils/rds-resources/utils';
 
 describe('check for unsupported RDS directives', () => {
-  const modelToDatasourceMap = new Map();
-  modelToDatasourceMap.set('Post', { dbType: 'MYSQL' });
-  modelToDatasourceMap.set('Tag', { dbType: 'DYNAMODB' });
+  const dataSourceStrategies: Record<string, ModelDataSourceStrategy> = {
+    Post: {
+      name: 'mysqlstrategy',
+      dbType: MYSQL_DB_TYPE,
+      dbConnectionConfig: {
+        databaseNameSsmPath: '/databaseNameSsmPath',
+        hostnameSsmPath: '/hostnameSsmPath',
+        portSsmPath: '/portSsmPath',
+        usernameSsmPath: '/usernameSsmPath',
+        passwordSsmPath: '/passwordSsmPath',
+      },
+    },
+    Tag: DDB_DEFAULT_DATASOURCE_STRATEGY,
```

Comment:
similar to the default strategy for DDB, can we add one for SQL strategy as well? This will save us from copying this in other tests that use the same.